dropwizard-cassandra/src/main/java/smartthings/dw/cassandra/CassandraHealthCheck.java

```diff
@@ -10,14 +11,14 @@
 public class CassandraHealthCheck extends NamedHealthCheck {
 	private final static Logger LOG = LoggerFactory.getLogger(CassandraHealthCheck.class);
 	private final Session session;
-	private final String validationQuery;
-	private final boolean validationQueryIdempotence;
+	private final PreparedStatement validationQuery;
```

Comment:
What do you think about switching this to a SimpleStatement?
https://docs.datastax.com/en/drivers/java/3.2/com/datastax/driver/core/SimpleStatement.html

We should then be able to drop all the strange validation logic around idempotence.  Just default the SimpleStatement to the default query + idemptent, and in constructor wrap it in a PreparedStatement.

Then if folks want to override it, they can have complete control over the statement object being executed rather than just the fields with expose piece meal.

Reply:
`SimpleStatement` makes sense, so I have made the change.

But I am not sure I understand how a `SimpleStatement` would make a difference with regards of configuring the idempotency value of the query. It seems the API would be the same, from the docs:

```
Statement s = new SimpleStatement("SELECT * FROM users WHERE id = 1");
s.setIdempotent(true);
```